tests/Unit/Evolution/Systems/GeneralizedHarmonic/TestFunctions.py

```diff
@@ -196,3 +195,199 @@ def two_index_constraint(d_gauge_function, spacetime_normal_one_form,
     return constraint
 
 # End test functions for two-index constraint
+
+
+# Begin tests for Test_DampedHarmonicGaugeQuantities.cpp
```

Comment:
No, please don't request changes that go against our documentation: https://spectre-code.org/writing_unit_tests.html

Reply:
I think with #1199 and merging #1204 we decided to allow splitting `TestFunctions.py` into multiple files if it makes sense. That's how I understand it at least. And in this case it makes a lot of sense to split a section that is 200 lines long (half the entire file) and independent of the other functions in that file off into a separate file.

Reply:
If you would like to see this changed please feel free to bring it up at a spectre call, put in a PR updating the documentation, and changing any files that need to be changed. I don't think we should be telling people to do things that go against the documentation. That is a very confusing thing to do.

Reply:
I don't care enough about this to try to push a docs PR through if there's still disagreemeent. I was under the impression we made a decision on issue #1199 to allow splitting `TestFunctions.py` going forward (see your comment in that same issue). We have since then done just that, e.g. `KomissarovShock.py`, `Lorentzian.py` (which were of course written by myself). I don't insist on splitting this file (as you say, it's not in the guidelines), but I do suggest it.

Reply:
I'm not saying I disagree with the change. I'm still fine with the change, I'm disagreeing with requesting it as a change. If someone wants to split the files I'm completely fine with that, it's just that the original comment in this thread reads as a required request for approval, not a suggestion.

Reply:
I'm getting mixed signals here :smile:

Reply:
I've moved these tests to `DampedHarmonicGaugeQuantities.py`